tox.ini

```diff
@@ -10,6 +10,7 @@ deps=
   moto<1.0
   HTTPretty==0.8.10
   nose<2.0
+  docker
```

Comment:
Do you want to specify a specific version?

Reply:
yes thanks, well spotted!